### PR TITLE
Change in Alex workflow

### PR DESCRIPTION
**Pull request by venkuppu-chn**
Problem : Alex is not getting triggered on raising a new PR
 
Solution: Changes are done so that Alex event will get triggered whenever the PR is opened (pull_request). 

Kindly refer Alex documentation https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/584778705/Alex+Workflow.

Signed-off-by: Venkatesh K <venkatesh.k@seagate.com>

### CODE REVIEW

**Comment by knekrasov** [comment]
@venkuppu-chn Thank you for the PR!

Can you please specify in the PR description what problem does your patch solve?

And please also reword your commit message to the following format:

```
<JIRA ID if it exists>: <Short problem statement>

<Detailed problem description>
Solution: <Solution that the commit suggests>

<Signed-off-by: ...>
```

**Comment by venkuppu-chn** [comment]
@knekrasov The change is not much, we changed the event trigger pull_request opened event instead cla-signed label.

**Comment by knekrasov** [comment]
> The change is not much, we changed the Alex event trigger to pull_request opened event instead cla-signed label.

@venkuppu-chn I understand that some of the labels were changed. But I'm not quite in the context of this bot and what this change will mean to the component. I don't think it is fair to require every team member to be deeply in context of every particular fix. Instead, can you summarize the problem and the essence of the solution like I [showed in the comment above](https://github.com/Seagate/cortx-hare/pull/1785#issuecomment-916044932)? Thank you!

**Comment by venkuppu-chn** [comment]
> > The change is not much, we changed the Alex event trigger to pull_request opened event instead cla-signed label.
> 
> @venkuppu-chn I understand that some of the labels were changed. But I'm not quite in the context of this bot and what this change will mean to the component. I don't think it is fair to require every team member to be deeply in context of every particular fix. Instead, can you summarize the problem and the essence of the solution like I [showed in the comment above](https://github.com/Seagate/cortx-hare/pull/1785#issuecomment-916044932)? Thank you!

I have modified the PR description as suggested.

**Comment by knekrasov** [comment]
@venkuppu-chn sorry to be annoying, but can you reword the commit message respectfully?

Suggestion:
```
Problem: Alex bot not triggered when a PR is opened

Solution: fix the labels in alexbot configuration file.

Signed-off-by: ...
```

**Comment by venkuppu-chn** [comment]
> @venkuppu-chn sorry to be annoying, but can you reword the commit message respectfully?
> 
> Suggestion:
> 
> ```
> Problem: Alex bot not triggered when a PR is opened
> 
> Solution: fix the labels in alexbot configuration file.
> 
> Signed-off-by: ...
> ```

Done.

**Comment by suvratjoshi** [comment]
@venkateshvb 
Thanks for making changes to PR description and commit message, so that problem and solution are pretty easy to understand.
Approving.
@mssawant , Can we merge this PR today. Thanks!

**Review by suvratjoshi — APPROVED** [review]

**Review by mssawant — APPROVED** [review]
